users.go

```diff
@@ -437,3 +437,152 @@ func (s *UsersService) UnblockUser(user int) error {
 		return fmt.Errorf("Received unexpected result code: %d", resp.StatusCode)
 	}
 }
+
+// EMail represents an EMail.
+//
+// GitLab API docs: https://doc.gitlab.com//help/api/users.md#list-emails
+type EMail struct {
+	ID    int    `json:"id"`
+	EMail string `json:"email"`
+}
+
+// ListEMails gets a list of currently authenticated user's EMails.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/api/users.html#list-emails
+func (s *UsersService) ListEMails() ([]*EMail, *Response, error) {
+	req, err := s.client.NewRequest("GET", "user/emails", nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var k []*EMail
```

Comment:
The `k` stood for `key` in the `SSH` calls, so it should be `e` here to keep things consistent (applies to all instances of course)